pages/index.vue

```diff
@@ -138,3 +153,21 @@ export default Vue.extend({
   }
 });
 </script>
+
+<style lang="scss" scoped>
+@import '~vuetify/src/styles/styles.sass';
+.home-header-margin {
+  margin-top: -56px;
+}
+
+@media #{map-get($display-breakpoints, 'md-and-up')} {
+  .home-header-margin {
+    margin-top: -64px;
+  }
+}
```

Comment:
I think it'd be better to move this to the `HomeHeader.vue`. Because currently, if no items are returned, the searchbar and Libraries text will overlap